[2025-05-16T08:05:23.214+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: brewery_pipeline.extract_bronze_data manual__2025-05-16T08:04:47.813585+00:00 [queued]>
[2025-05-16T08:05:23.222+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: brewery_pipeline.extract_bronze_data manual__2025-05-16T08:04:47.813585+00:00 [queued]>
[2025-05-16T08:05:23.222+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2025-05-16T08:05:23.240+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_bronze_data> on 2025-05-16 08:04:47.813585+00:00
[2025-05-16T08:05:23.244+0000] {standard_task_runner.py:57} INFO - Started process 44 to run task
[2025-05-16T08:05:23.247+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'brewery_pipeline', 'extract_bronze_data', 'manual__2025-05-16T08:04:47.813585+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/brewery_pipeline.py', '--cfg-path', '/tmp/tmpm8908_ub']
[2025-05-16T08:05:23.249+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask extract_bronze_data
[2025-05-16T08:05:23.267+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:193 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-16T08:05:23.291+0000] {task_command.py:416} INFO - Running <TaskInstance: brewery_pipeline.extract_bronze_data manual__2025-05-16T08:04:47.813585+00:00 [running]> on host 477fbce01ca2
[2025-05-16T08:05:23.342+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mateus' AIRFLOW_CTX_DAG_ID='brewery_pipeline' AIRFLOW_CTX_TASK_ID='extract_bronze_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-16T08:04:47.813585+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-16T08:04:47.813585+00:00'
[2025-05-16T08:05:23.398+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/brewery_pipeline.py", line 33, in extract_bronze_data
    with open(output_path, 'w') as f:
PermissionError: [Errno 13] Permission denied: '/opt/airflow/data/bronze/breweries_raw.json'
[2025-05-16T08:05:23.404+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=brewery_pipeline, task_id=extract_bronze_data, execution_date=20250516T080447, start_date=20250516T080523, end_date=20250516T080523
[2025-05-16T08:05:23.427+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 6 for task extract_bronze_data ([Errno 13] Permission denied: '/opt/airflow/data/bronze/breweries_raw.json'; 44)
[2025-05-16T08:05:23.458+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-05-16T08:05:23.473+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
